packages/flutter/lib/src/material/data_table.dart

```diff
@@ -109,8 +109,8 @@ class DataRow {
     this.onSelectChanged,
     this.color,
     required this.cells,
-  }) : assert(cells != null),
-       key = ValueKey<int?>(index);
+  })   : assert(cells != null),
+        key = ValueKey<int?>(index);
```

Comment:
The "key" doesn't seem to align with the ":".

Reply:
Please check now